src/sentry/snuba/metrics/extraction.py

```diff
@@ -1513,7 +1514,7 @@ def _get_satisfactory_threshold_and_metric(project: Project) -> tuple[int, str]:
 
     if len(result) == 0:
         # We use the default threshold shown in the UI.
-        threshold = APDEX_THRESHOLD_DEFAULT
```

Comment:
This will change the spec json but not the query_hash for specs (because this is a default value filled in for `_arguments`)? Are we fine with that change? I don't see a test in `test_extraction.py` for apdex with a default value (eg. apdex()), can we add one and check the conditions before and after?